### ISSUE TITLE
jobtap: FileNotFoundError for loading the same jobtap plugin twice (does exist)

### ISSUE THREAD

**Issue by wihobbs** [flux-framework/flux-core#5545]
When loading the same jobtap plugin twice, flux raises [an `ENOENT` error on the python side](https://github.com/flux-framework/flux-core/blob/ebf391f5df6315222f16b6a3442d471e212fc183/src/cmd/flux-jobtap.py#L40):

```
ƒ(s=1,d=0) fluxuser@aef8ee741887:/home/hobbs17/flux-notify/src$ flux jobtap load /home/hobbs17/flux-notify/src/notify.so 
ƒ(s=1,d=0) fluxuser@aef8ee741887:/home/hobbs17/flux-notify/src$ flux jobtap load /home/hobbs17/flux-notify/src/notify.so 
flux-jobtap: ERROR: /home/hobbs17/flux-notify/src/notify.so not found
```

Interestingly, in `flux dmesg`, the corresponding services to that plugin register twice and do not raise errors (from what I can see):
```
2023-11-08T19:45:20.556655Z job-manager.debug[0]: jobtap plugin notify registered method job-manager.notify.start
2023-11-08T19:45:20.556685Z job-manager.debug[0]: jobtap plugin notify registered method job-manager.notify.remove-job
2023-11-08T19:45:20.556687Z job-manager.debug[0]: jobtap plugin notify registered method job-manager.notify.disconnect
2023-11-08T19:45:21.522782Z job-manager.debug[0]: jobtap plugin notify registered method job-manager.notify.start
2023-11-08T19:45:21.522799Z job-manager.debug[0]: jobtap plugin notify registered method job-manager.notify.remove-job
2023-11-08T19:45:21.522802Z job-manager.debug[0]: jobtap plugin notify registered method job-manager.notify.disconnect
ƒ(s=1,d=0) fluxuser@aef8ee741887:/home/hobbs17/flux-notify/src$ 
```

**Comment by grondo** [comment]
This was fixed by #5537.

**Comment by wihobbs** [comment]
Oh, shoot! Sorry.
